react/features/recent-list/components/RecentList.native.js

```diff
@@ -82,13 +84,15 @@ class RecentList extends AbstractRecentList<Props> {
         const recentList = toDisplayableList(_recentList, t, _defaultServerURL);
 
         return (
-            <NavigateSectionList
-                disabled = { disabled }
-                onLongPress = { this._onLongPress }
-                onPress = { this._onPress }
-                renderListEmptyComponent
-                    = { this._getRenderListEmptyComponent() }
-                sections = { recentList } />
+            <View style = { disabled ? styles.recentListDisabled : styles.recentList }>
```

Comment:
Doesn't a similar thing need to be applied for the calendar tab?

Reply:
I applied it.